### DISCUSSION TITLE
Bindings to other languages?

### DISCUSSION THREAD

**Discussion by aryajur** [alandefreitas/matplotplusplus#379] [Q&A]
Hi,
        I have been using MathGL though Lua and saw matplot++. The quality of the graphs look really nice. I am wondering if there are any bindings for matplot++?
        Also when going through the install section I see gnuplot is required. Why is that required?

**Comment by alandefreitas** [comment]
I don't really know of many bindings. I think languages tend to have their own tools for that and that was just missing in C++ because of historical reasons related to the ecosystem.

Yes. Gnuplot is a dependency. The backend uses gnuplot. The original idea was to have multiple backends and make it very easy to extend but we ended up not having the time to implement that.
